plugins/DiskSize/migrations/001_migrations.rb

```diff
@@ -8,7 +8,10 @@
       foreign_key   :node_resource
       Time          :created, null: false
       String        :node
-      Integer       :value
+      Integer       :disk_count
```

Comment:
This rename makes sense from a plugin perspective, but if all measurement tables use the same field name for their measured value, then it will be easier to write utility methods to work with report data in a generic way. So I vote for using 'value', or another generic name for the measurement field